.travis.yml

```diff
@@ -0,0 +1,6 @@
+sudo: false
```

Comment:
There's no harm in being explicit. Hence the reason all of our rules are explicitly defined. :P